tests/param_tests.cpp

```diff
@@ -66,4 +68,12 @@ TEST_CASE("Percent and BiPolar Percent")
         REQUIRE(*(p.valueFromString("440", ems)) == 0);
         REQUIRE(*(p.valueFromString("220", ems)) == -12);
     }
-}
\ No newline at end of file
+
+    SECTION("Default Constructors")
```

Comment:
oh i love this style of test and hadn't even thought of it. Very clever. Thank you!